src/main/resources/assets/amunra/lang/en_US.lang

```diff
@@ -138,11 +138,11 @@ item.baseItem.dockGangway.name=Dock Gangway
 item.schematic.schematicShuttle.name=Shuttle Schematic
 item.baseItem.darkShard.name=Dark Matter Shard
 item.baseItem.naniteCluster.name=Nanite Cluster
-item.baseItem.naniteCluster.description=These nanites seem inactive, but still functional. Perhaps reprogramming them?
+item.baseItem.naniteCluster.description=These nanites seem inactive, but still functional.
 item.baseItem.naniteControl.name=Nanite Control Module
 item.baseItem.mummyDust.name=Mummy Dust
 
-item.porcodonMeat.description=Fuel Source. Eating not recommended.
+item.porcodonMeat.description=The deadly methane dose of this meat makes it un-edible, but it could be used as furnace fuel.
```

Comment:
Now that I read it, this line feels incredibly wrong. The current one is fine enough :)